terms/roccurves_pb.tex

```diff
@@ -0,0 +1,88 @@
+\chapter{Receiver Operating Characteristic ROC Curves}
+\label{ch:roc-curves}\index{roc-curves}
+
+\section{Introduction}
+A Receiver Operating Characteristic (ROC) curve is a graphical plot which presents the performance of a binary classifier when the discrimination cut-off, or threshold\index{roc-curves!threshold}, is varied. The ROC curves and associated analysis, such as area under the curve, comparing multiple ROC curves and finding an optimal threshold, have proved very useful analysis tools in many fields of research such as medical diagnosis and disease management. They have been found to be a more appropriate technique for evaluating diagnostic and predictive accuracy.
+
+Essentially for creating and plotting the ROC curve one needs to compute the true positive rate (TPR)\index{roc-curves!true positive rate} and the false positive rate (FPR)\index{roc-curves!false positive rate} at various threshold settings and plot them in a dimensional space. In machine learning the TPR is referred to as the sensitivity\index{roc-curves!sensitivity} or recall which measures the proportion of actual positives that are correctly classified. The true negative rate (TNR)\index{roc-curves!true negative rate}, or specificity\index{roc-curves!specificity}, measures the proportion of actual negatives that are correctly identified as such. These measures can be easily summarised in Table 1 below.
+
+\begin{table}[]
```

Comment:
this table breaks into the margin and overlays the caption ... there is an example of an environment to use the whole page (see fullwidth).  Also see the tufte book example in the subdirectory.